tests/Pest/ManagerTest.php

```diff
@@ -118,6 +120,16 @@
         ->toContain('<link rel="canonical" href="http://localhost" />');
 });
 
+test('canonical url accepts origin', function () {
+    $this->get('/testing/5');
+
+    seo()->withUrl('https://foo.com');
+
+    expect(meta())
+        ->toContain('<meta property="og:url" content="https://foo.com/testing/5" />')
+        ->toContain('<link rel="canonical" href="https://foo.com/testing/5" />');
```

Comment:
Can you link MDN docs or something where I can see that both `og:url` and `canonical` should be the canonical link?

Reply:
Not really sure why you're asking me this. The package is already setting og:url and canonical to the value of the current url. All my pull request does is give users the option to pass a value instead.

https://github.com/archtechx/laravel-seo/blob/f6fd5f4a551313926acf991f0f225ae6c3199cd1/resources/views/components/meta.blade.php#L31-L34